include/motor-control/core/brushed_motor/brushed_motor_interrupt_handler.hpp

```diff
@@ -201,7 +201,7 @@ class BrushedMotorInterruptHandler {
         } else if (estop_triggered()) {
             in_estop = true;
             cancel_and_clear_moves(can::ids::ErrorCode::estop_detected);
-        } else if (hardware.has_cancel_request()) {
+        } else if (hardware.has_cancel_request().code != 0U) {
```

Comment:
this code should definitely be comparing enum values; ideally the `CancelRequest` struct will have the enum values in it but if not let's do this compare in a static cast